### PR TITLE
Mostly complete new actor guide

### PR DESCRIPTION
**Pull request by autodidaddict**
New actor creation guide.

@brooksmtownsend My guide gets a little shaky when I start illustrating the REPL commands for starting an actor. Please make sure you take a close look at those and, if you can produce any kind of sample output from the REPL in response to those commands, that would be fantastic.

### CODE REVIEW

File: content/app-dev/create-actor/run.md [hunk 1]
```diff
@@ -5,4 +5,59 @@ weight: 5
...
+Next, we'll need to start the HTTP Server. Fortunately, the wasmCloud official HTTP server capability provider is published in Azure Container Registry, so you can start it with the following REPL command:
+
+```
+start provider https://wasmcloud.azurecr.io/http_server:0.15.0
+```
+
+With both the provider and the actor running, the only thing left to do is _link_ the two. This provides a set of configuration values that is unique to one actor's use of a provider. Type the following command to link your actor with the provider:
+
+```
+link ... ???? ??? ??
```
Comment (brooksmtownsend) [inline content/app-dev/create-actor/run.md:53]: This should be
```
ctl link (ACTOR_MODULE_KEY) VAG3QITQQ2ODAOWB5TTQSDJ53XK3SHBEIFNK4AYJ5RKAX2UNSCAPHA5M wasmcloud:httpserver PORT=8080
```
Perhaps we can also throw in a snippet like:

If you had your module key autogenerated via `wash` when you ran `make`, you can find that key by running the following command in the REPL:
```
claims inspect --insecure localhost:5000/newactor:v1
```
Under the `Module` section, you'll see a 56 character public key starting with **M**. That is your actor's module key that you can use with the previous `link` command.

File: content/app-dev/create-actor/run.md [hunk 2]
```diff
@@ -5,4 +5,59 @@ weight: 5
...
+```
+
+You should see in the log output that the actor configuration was passed and you should see an HTTP server starting on port 8080. Now you can type the following in the wash REPL to curl your newly started endpoint:
```
Comment (brooksmtownsend) [inline content/app-dev/create-actor/run.md:56]: They will need to type this in a separate terminal window to use `curl`

File: content/app-dev/create-actor/handlers.md [hunk 3]
```diff
@@ -5,18 +5,42 @@ weight: 4
...
-   {{% /tab %}}
- {{% /tabs %}}
+```toml
+serde_json = "1.0.59"
+actor-http-server = { git = "https://github.com/wasmcloud/actor-interfaces", branch = "main", features = ["guest"]}
+```
 
-TBD
+In this case we're using a git reference for the actor interface,
+ but you'll want to use the [crates.io](https://crates.io/crates/actor-http-server) version for anything more than experimentation.
```
[the remark this answers is not in the document]
Comment (brooksmtownsend) [reply content/app-dev/create-actor/handlers.md:18]: (maybe we file an issue for this in this repo, to comb over the docs for any `git` references, `wascc` snippets, etc?)

**Review by brooksmtownsend — CHANGES_REQUESTED** [review]
A couple of changes requested for the specific commands for the REPL

File: content/app-dev/create-actor/handlers.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (autodidaddict) [reply content/app-dev/create-actor/handlers.md:18]: makes sense to file that issue after we have the docs in place. I'd rather have something in place using git refs than having nothing there.

File: content/app-dev/create-actor/run.md [hunk 4]
```diff
@@ -5,4 +5,59 @@ weight: 5
...
+
+### Push Your Actor
+
+Next you'll want to push your actor to the local registry. If you haven't configured authentication or any other restrictions, you should be able to use the following `wash` command to push it to the registry:
+
+```
+wash reg push --insecure localhost:5000/(image):(tag) ./target/wasm32-unknown-unknown/debug/(actor).wasm
+```
+
+For example, you might use `newactor` and `v1` as the image and tag respectively, giving you a local registry URL (you'll need to remember this) of `http://localhost:5000/newactor:v1`. Make sure the path you supply here is to the _signed_ version of your actor's WebAssembly module.
```
[the remark this answers is not in the document]
Comment (autodidaddict) [reply content/app-dev/create-actor/run.md:30]: Fixed.

**Review by brooksmtownsend — APPROVED** [review]

**Pull request merged**